internal/policy/filter.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 The Flux authors
```

Comment:
Yes, I didn't bother with this because I already had this laying around since last year. I think 2021 is better for this particular file because it was pretty much rewritten entirely today.